### DISCUSSION TITLE
The error of withBinaryFile: does not exist (No such file or directory)

### DISCUSSION THREAD

**Discussion by stklj** [jgm/pandoc#9572] [Q&A]
I want to use pandoc to transfer the tex file to docx file. I have set the environment path and 
 ensured the existence of my tex file but it still doesn't work.

**Comment by jgm** [comment]
What pandoc command did you use, exactly? Where are the relevant files?  What is the precise error message you get?

**Comment by denismaier** [comment]
Can you try in Windows cmd prompt? Also, it try relative paths

pandoc ./file.md -o file.pdf

Gesendet von Outlook für Android<https://aka.ms/AAb9ysg>
________________________________
